README.md

```diff
@@ -33,6 +33,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Rust](#rust)
 - [Scala](#scala)
 - [TypeScript](#typescript)
+- [V](#v)
```

Comment:
@Delta456 I was wondering if we could add the `v-lang` here as well. We'd need to label an issue as `good first issue` and then link here. I think that'd be great :)

Reply:
Yeah sure

Reply:
Could you label a couple of issues so we could add it?